modules/ROOT/pages/provisioning-applehv.adoc

```diff
@@ -0,0 +1,56 @@
+= Provisioning Fedora CoreOS on AppleHV
+
+This guide shows how to provision new Fedora CoreOS (FCOS) instances on MacOS using https://github.com/crc-org/vfkit[vfkit].
+
+
+== Prerequisites
+
+Before provisioning an FCOS machine, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You will also need to build or acquire https://github.com/crc-org/vfkit[vfkit].  Prebuilt binaries are also available from its https://github.com/crc-org/vfkit/releases/tag/v0.6.0[releases] page.  Vfkit, like QEMU, has many options which are outside the scope of this provisioning example.  Please consider reading their https://github.com/crc-org/vfkit/tree/main/doc[documentation].
+
+== Booting a new VM on MacOS
+
+This section shows how to boot a new VM with vfkit. Vfkit is known to work with both Intel and Apple Silicon based Macs.
+
+=== Fetching the AppleHV image
+
+Fetch the latest image suitable for your target stream (or https://fedoraproject.org/coreos/download/[download and verify] it from the web).  Remember to download the approach image based on the architecture of your Mac.  Once downloaded, you will also need to decompress the image.
+
+=== Setting up a new VM
+
+Vfkit is not a stateful virtual machine framework. You simply need to run the vfkit binary to start a virtual machine. The following command line will launch a VM with:
+
+* 2 virtual CPUs
+* 2 GB of memory
+* a network device that will receive a IP address from Vfit
+* a GUI console with keyboard and mouse support
+
+.Launching FCOS with Vfkit
+[source, bash]
+----
+IGNITION_CONFIG="/path/to/example.ign"
+IMAGE="/path/to/image.qcow2"
+
+./vfkit --cpus 2 --memory 2048 \
+  --bootloader efi,variable-store=efi-variable-store,create \
+  --device virtio-blk,path=${IMAGE} \
+  --device virtio-net,nat \
```

Comment:
maybe you can hardcode a mac address here so that the VM IP is always the same? ok with me if you consider this outside of the scope of this document.

Reply:
i was trying to make it dead simple.  I dont object.  See what others say

Reply:
Does a specific MAC make the configured IP deterministic?

I lean more towards not harcoding a MAC address here..

Reply:
not deterministic imho

Reply:
It's more deterministic in the sense that you will always get the same mac address every time you run the VM. But I agree it makes the example more complicated, and it's not really important.